pilot/pkg/networking/core/v1alpha3/route/retry/retry.go

```diff
@@ -83,6 +94,30 @@ func ConvertPolicy(in *networking.HTTPRetry) *route.RetryPolicy {
 	if in.PerTryTimeout != nil {
 		out.PerTryTimeout = util.GogoDurationToDuration(in.PerTryTimeout)
 	}
+
+	if in.RetryRemoteLocalities != nil && in.RetryRemoteLocalities.GetValue() {
+		// UpdateFrequency indicated how often to update the priority.
+		// for now we set a default value: 2 for it.
+		previousPriorities := &previouspriorities.PreviousPrioritiesConfig{
+			UpdateFrequency: int32(2),
+		}
+
+		data, err := proto.Marshal(previousPriorities)
```

Comment:
Can you do something like this instead? https://github.com/istio/istio/blob/44f62301c5935d686b303f2ae971f9a2e769db55/pilot/pkg/networking/core/v1alpha3/listener.go

Reply:
Done